### PR TITLE
fix(replays): add default user_id  to first replay_event url sent

### PR DESCRIPTION
**Pull request by JoshFerge**
events need user_id to be considered valid for analytics.

### CODE REVIEW

File: src/sentry/analytics/events/first_replay_sent.py [hunk 1]
```diff
@@ -8,6 +8,7 @@ class FirstReplaySentEvent(analytics.Event):
         analytics.Attribute("organization_id"),
         analytics.Attribute("project_id"),
         analytics.Attribute("platform", required=False),
+        analytics.Attribute("default_user_id", required=False),
```
Comment (scefali) [inline src/sentry/analytics/events/first_replay_sent.py:11]: @JoshFerge It had to be literally user_id or it won’t work. Also, I added a new field called default_owner_id that doesn’t raise an exception

Comment (JoshFerge) [reply src/sentry/analytics/events/first_replay_sent.py:11]: i see on `FirstTransactionSentEvent` just has `default_user_id`?

Comment (JoshFerge) [reply src/sentry/analytics/events/first_replay_sent.py]: regardless, went ahead and updated

Comment (scefali) [reply src/sentry/analytics/events/first_replay_sent.py]: @JoshFerge ok yes there’s a way to use a different ID field but you have to update the ETL repo and add the mapping and also we use the default owner as the user_id for almost every other backend event. Better for consistency’s sake:

**Review by scefali — APPROVED** [review]

**Pull request merged**